src/game_character.cpp

```diff
@@ -86,6 +86,8 @@ bool Game_Character::IsLandable(int x, int y) const
 
 	if (GetThrough()) return true;
 
+	if (IsFlying()) return true;
```

Comment:
> Airship can jump and land anywhere 

Just a comment: It is confusing that "IsLandable" is not about Airship but Jump landing :D

Reply:
This mess is getting cleaned up in #1628